Changes to pandas to match branding (#2118)
fishtown-analytics/docs.getdbt.com: website/docs/docs/building-a-dbt-project/building-models/python-models.md

#### Code reuse

> Currently, Python functions defined in one dbt model cannot be imported and reused in other models. This is something we'd like dbt to support. There are two patterns we're considering:

Reviewer: ```suggestion
Currently, you cannot import or reuse Python functions defined in one dbt model, in other models.  This is something we'd like dbt to support. There are two patterns we're considering:
```

Author: Good catch, I missed this previously.  Thank you!

> 1. Creating and registering **"named" UDFs**. This process is different across data platforms and has some performance limitations. (Snowpark does support ["vectorized" UDFs](https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-batch.html): pandas-like functions that can be executed in parallel.)

Reviewer: switch to active voice
```suggestion
1. Creating and registering **"named" UDFs**. This process is different across data platforms and has some performance limitations. (Snowpark does support ["vectorized" UDFs](https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-batch.html): pandas-like functions that you can execute in parallel.)
```
